plugin/Makefile

```diff
@@ -17,37 +17,65 @@
 # Eventually we will throw away deb/rpm build from ../vmdk_plugin/Makefile and will use only
 # managed plugin
 
-# Allow user name for dockerhub (and first components of the plugin name) to be supplied in env
-# Otherwise, grab it from current user (`whoami`)
-DOCKER_HUB_USER ?= $(shell whoami)
+# Makefile by default build a name in this format (example) 'msterin/docker-volume-vsphere:0.13-dev'
+#
+# You can use the following environment vars to change the behavior :
+#
+# DOCKER_HUB_USER - name of user and dockerhub repo to use. Default is `whoami`
+#                   Please 'docker login' there is you want to push
+# VERSION_TAG     - How you want the version to look. Default is "current git tag +1 "
+# EXTRA_TAG       - additional info you want in tag. Default is "-dev"
+#
+# To check resuting settings use  "make info"
+# examples:
+#   make
+#          --> resulting name (for my build as of 3/17/17)  msterin/docker-volume-vsphere:0.13-dev
+#
+# 	DOCKER_HUB_USER=vmware EXTRA_TAG= VERSION_TAG=latest make
+#           --> resulting name vmware/docker-volume-vsphere:latest
+#
+#   DOCKER_HUB_USER=cnastorage EXTRA_TAG=-CI make
+#           --> resulting name cnastorage/docker-volume-vspehere:0.13-CI
+#
 
 # grab the latest commit SHA and related tag from git so we can construct plugin tag
 GIT_SHA := $(shell git rev-parse --revs-only --short HEAD)
 GIT_TAG := $(shell git describe --tags --abbrev=0 $(GIT_SHA))
 
-# Allow PLUGIN_TAG to be supplied via env, or use the latest git tag.
-# use EXTRA_TAG for adding suffixes like "-CI" or "-RELEASE" (otional)
-PLUGIN_TAG ?= $(GIT_TAG)$(EXTRA_TAG)
+# Allow these vars to be suplied in environment
+DOCKER_HUB_USER ?= $(shell whoami)
+# "git tag + 1" , so "0.11" becomes "0.12"
+VERSION_TAG ?= $(shell echo $(GIT_TAG) | awk -F. '{printf ("%d.%d", $$1, $$2+1)}' )
+EXTRA_TAG ?= -dev
+
+# final tag
```

Comment:
I would like to avoid enumerting all possibilities as I do not know how we will name the release. But an example is already there:
```
# DOCKER_HUB_REPO=vmware EXTRA_TAG= VERSION_TAG=latest make
#           --> resulting name vmware/docker-volume-vsphere:latest
```